index.html

```diff
@@ -61,10 +80,11 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
   
 
     <!-- work one section -->
-
+    <section id="Portfolio">
      <section class="card-works works">
       <div class="image ">
-        <img src="images/Snap.png" alt="Card1"/>
+        <img class="desktop-image" src="images/Snap.png" alt="Project"/>
+        <img class="mobile-image" src="images/Snapshoot1.png" alt="Project"/>
       </div>
```

Comment:
- [ ]  Kindly comment these cards from your html file and generate your cards dynamically from javascript using an array of objects :+1: 

  **Note**: You can see this [javascript file](https://github.com/PeteUgwu/Portfolio-set-up/pull/11/files#diff-ed3ee7e0beea2498ff3b8ca85973d122fc6fa3d585d62b5807ec034d0cf076b3) to get an understanding how to implement both of these requirements:+1: